src/ansys/fluent/core/workflow.py

```diff
@@ -1707,14 +1717,6 @@ def __call__(self):
         """Delegate calls to the underlying workflow."""
         return self._workflow()
 
-    def _attr_from_wrapped_workflow(self, attr):
-        try:
-            result = getattr(self._workflow, attr)
-            if result:
-                return result
-        except AttributeError:
-            pass
-
 
 class ReadOnlyObject:
     """Removes set_state() to implement read-only behaviour."""
```

Comment:
```suggestion
    """Removes ``set_state()`` to implement read-only behavior."""
```
U.S. English should be used. Is ``set_state()`` a method. Again, there is no noun to provide clarity.